h2/src/main/org/h2/message/DbException.java

```diff
@@ -345,6 +356,11 @@ private static JdbcSQLException getJdbcSQLException(int errorCode,
         return new JdbcSQLException(message, null, sqlstate, errorCode, cause, null);
     }
 
+    private static JdbcSQLException getJdbcSQLException(String sqlstate, String message) {
+        // do not translate as sqlstate is arbitrary : avoid "message not found"
+        return new JdbcSQLException(message, null, sqlstate, 0, null, null);
```

Comment:
please inline this method into fromUser

Reply:
OK, will do. I chose to put it there so that all calls to the JdbcSQLException constructor were in the the same place.